aten/src/ATen/native/README.md

```diff
@@ -286,6 +286,32 @@ In addition to the backends above, we also support the keywords:
     registered to `Math` key should be plain mathematical composition of other
     `at::` functions and support training and inference for any backend.
```

Comment:
Here I think we should add a paragraph like:

`DefaultBackend` and `Math` keys act as defaults: for example, you can specify a custom kernel for a particular backend using a backend-specific dispatch key, and use `DefaultBackend` or `Math` to specify a generic kernel for the others.

Reply:
**Also** these aren't part of your PR, but it's worth fixing them while you're here:

line 271-272: "depending on whether or not you have CPU or CUDA tensors" should become something like "depending on which backend the passed tensors belong to" 

line 281: this link is out of date, it points to the middle of some of the new codegen. We should either update it or just give the file and a keyword to search on or something